src/components/Modals/ShortcutsModal/ShortcutsModal.tsx

```diff
@@ -34,97 +34,114 @@ export default class ShortcutsModal extends React.PureComponent<Props> {
           <div className="title">{t('shortcuts_modal.title')}</div>
 
           <div className="shortcut-list">
-            <div className="subtitle">{t('shortcuts_modal.basic_controls.title')}</div>
+            <div className="subtitle">{t('shortcuts_modal.basic_controls')}</div>
 
             <div className="shortcuts">
               <div className="shortcut">
-                <div className="name">{t('shortcuts_modal.basic_controls.shortcut_reference')}</div>
+                <div className="name">{t('shortcuts_modal.shortcut_reference')}</div>
                 <div className="keybinding">
                   <Chip text="?" />
                 </div>
               </div>
 
               <div className="shortcut">
-                <div className="name">{t('shortcuts_modal.basic_controls.undo')}</div>
+                <div className="name">{t('shortcuts_modal.move_camera')}</div>
                 <div className="keybinding">
-                  <Chip text={this.getMainShortcutKey()} />
-                  <span className="plus">+</span>
-                  <Chip text="z" />
+                  <Chip text="▲" />
+                  <Chip text="▼" />
+                  <Chip text="◄" />
```

Comment:
aren't these different? hehe

Reply:
?

Reply:
never mind, they looked narrow but it's github's font